bin/main.rb

```diff
@@ -1,2 +1,74 @@
 #!/usr/bin/env ruby
-puts 'Hello world!'
+
+puts 'INSTRUCTIONS'
+puts '---------------------------------------------------'
+puts 'A number represents each square spaces on the board'
+puts 'from 1 up to 9. To make a move specifies a number'
+puts 'in the range of possible numbers at each step into'
+puts 'the game. It\'s that simple :). Enjoy!!!'
+
+puts 'Introduce the name of the first player.'
+player_one = gets.chomp
+
+puts 'Introduce the name of the second player'
+player_two = gets.chomp
+start = 1
+
+# now i'll display the board and an array whith each available moves
+
+while start.positive?
+
+  # here a methode will be responsible to display the board after each player's move.
+  # the board will display after each player,s move except for the first time we start.
+
+  puts 'Player one\'s turn!'
+  player_one_turn = gets.chomp.to_i
+
+  # here instead of using the if that way i'll check if the number i got from the player
+  # input is included in the array of possible moves left i'll create later
+  # same for each player
+
+  if player_one_turn < 1 && player_one_turn > 9
+    puts 'Oups! you made a wrong move'
+    puts 'Player one\'s turn!'
+    player_one_turn = gets.chomp.to_i
+  end
+
+  puts '- - -'
+  puts '- - -'
+  puts '- - -'
```

Comment:
If you use the code to puts lines directly, it can not be modified to change when the user provides input. If you store these lines on an array, for example, you can easily add the logic to replace the '-' for a symbol 'X' or 'O' to indicate the move.